assignments/lambda-classes.js

```diff
@@ -1 +1,103 @@
-// CODE here for your Lambda Classes
+class Person{
+    constructor(baseStats){
+        this.name = baseStats.name;
+        this.age = baseStats.age;
+        this.location = baseStats.location;
+        this.gender = baseStats.gender;
+    }
+
+        speak(){
+            return `Hello myname is ${this.name}, I am from ${this.location}!`
+        }
+
+}
+
+class Instructor extends Person{
+    constructor (insStats){
+        super(insStats)
+        this.specialty = insStats.specialty;
+        this.favLanguage = insStats.favLanguage;
+        this.catchPhrase = insStats.catchPhrase;
+    }
+        demo(subject){
```

Comment:
The methods spacing here is a bit off. They should be inline with the constructor.